tiatoolbox/dataloader/wsireader.py

```diff
@@ -84,15 +85,148 @@ def slide_info(self):
         """
         raise NotImplementedError
 
-    def read_region(self, start_w, start_h, end_w, end_h, level=0):
+    def relative_level_scales(self, target_scale, units):
+        """
+        Calculate scale of each image pyramid level relative to the
+        given target scale and units.
+
+        Values > 1 indicate that the level has a larger scale than the
+        target and < 1 indicates that it is smaller.
+
+        Args:
+            target_scale (float): Scale to calculate relative to
+            units (str): Units of the scale. Allowed values are: mpp,
```

Comment:
Should this be written as Allowed values are: "mpp", "power", "level" and "baseline"